wrappers/services.go

```diff
@@ -93,70 +93,55 @@ func min(a, b int) int {
 }
 
 func formatCpuGroupSlice(grp *quota.Group) string {
-	if grp.CPULimit == nil {
```

Comment:
This part now got removed - and hence the tests got updated. I think it's harmless but just wanted to double check that it's okay that we now always enable CPU accounting even with CPULimit == nil ?

Reply:
Yea it's harmless, we also always MemoryAccounting and TasksAccounting set even though we don't set any limits.